cmd/thanos/query.go

```diff
@@ -222,14 +226,20 @@ func runQuery(
 		dns.ResolverType(dnsSDResolver),
 	)
 
+	staticStores, dynamicStores := dns.FilterStaticNodes(storeAddrs...)
```

Comment:
This is weird that filterstatic nodes function gives you both static and dynamic.. 🤔

Reply:
I agree. We should probably rename it to `FilterNodes` or something. WDYT?